### ISSUE TITLE
False Option warning on Windows

### ISSUE THREAD

**Issue by MikeMcC399** [bmish/eslint-doc-generator#523]
## Issue

`eslint-doc-generator` outputs the following message although the rule doc concerned includes an `Options` section:

> rule doc should have included one of these headers: Options, Config

This problem seems to affect Windows only. I could not reproduce it on Ubuntu `22.04.4` LTS for example.

## Versions

Operating System:       Windows 11
Node.js:                `20.12.2` LTS
eslint:                 `8.57.0`
eslint-doc-generator:   `1.7.0`

## Steps to reproduce

On Windows 11 with Node.js `20.12.2` LTS

```shell
git clone --branch show/doc-generation https://github.com/MikeMcC399/eslint-plugin-cypress --single-branch
cd eslint-plugin-cypress
npm ci
npx eslint-doc-generator
```

Note the error message

> `unsafe-to-chain-command` rule doc should have included one of these headers: Options, Config

View the file `docs/rules/unsafe-to-chain-command.md` and note that it includes a header `Options`:

```markdown
## Options

<!-- begin auto-generated rule options list -->

| Name      | Description                                                 | Type  | Default |
| :-------- | :---------------------------------------------------------- | :---- | :------ |
| `methods` | An additional list of methods to check for unsafe chaining. | Array | `[]`    |

<!-- end auto-generated rule options list -->
```

**Comment by bmish** [comment]
Thanks for the detailed report. I haven't been able to test on Windows yet but I suspect I have a fix in #524.

**Comment by bmish** [comment]
Let me know if https://github.com/bmish/eslint-doc-generator/releases/tag/v1.7.1 fixes this.

**Comment by MikeMcC399** [comment]
@bmish 

Same success for this issue as well 👍🏻 

Thank you very much for fixing this issue! I ran it on the production version of `eslint-plugin-cypress` and it's working perfectly now:

```text
git clone https://github.com/cypress-io/eslint-plugin-cypress
cd eslint-plugin-cypress
npm ci
npm install eslint-doc-generator@1.7.1 -D
npx eslint-doc-generator
```

No option warning appears anymore!
